TESTING.md

```diff
@@ -69,6 +69,38 @@ tests are finished. The command below will give you a shell after all tests are
 You can use it to inspect test logs or even modify test sources and run your tests
 manually following the method described above. For this purpose you can find test sources under `/var/tmp/tmt/run-*/packit-ci/discover/default/tests/`.
 
+### Running multi-host tests
+
+`tmt` cannot schedule multi-host tests yet. However, we can use `tmt` to do the necessary setup and then execute the test manually.
+
+For `Multihost/basic-attestation` test we need at least 3 test systems, i.e. 3 terminals.
+
+On terminal 1:
+```
+$ git clone https://github.com/RedHat-SP-Security/keylime-tests.git
+```
+
+On all terminals:
+```
+$ cd keylime-tests
```

Comment:
This is confusing. From the previous instruction, we only cloned the repo in terminal 01. Should we do the same for all the terminals, or should we simply create a `keylime-tests` directory in the other ones?

Reply:
All 3 terminals are on a local system so it is enough to clone the repo once and enter it in each terminal. I will rephrase it.

Reply:
Ahaa, now it makes sense. Inded, I misunderstood the beginning of the instructions. Here I had 3 different systems, then 3 terminals, one for each system :)

Reply:
Updated.